src/components/home/Popup.tsx

```diff
@@ -0,0 +1,114 @@
+/** @jsx jsx */
+import React from 'react';
+import { jsx, SxStyleProp } from 'theme-ui';
+
+import { IPost } from '../../utils/interface';
+import { theme } from '../../utils/theme';
```

Comment:
It's not needed to import `theme`

Reply:
I removed that but I guess I didn't push it in this commit. Weird